packages/common/src/audius-query/createApi.ts

```diff
@@ -294,6 +295,8 @@ const useCacheData = <Args, Data>(
   }, isEqual)
 }
 
+const requestBatcher = createRequestBatcher()
```

Comment:
do we want one request batcher for all apis vs one for each endpoint?

Reply:
Good thought, I think one is simpler because the requests are bucketed both on the endpoint and the arguments. So it's nice to have everything in one big map